storm-core/src/jvm/org/apache/storm/utils/DisruptorQueue.java

```diff
@@ -63,6 +63,18 @@
     private static final String PREFIX = "disruptor-";
     private static final FlusherPool FLUSHER = new FlusherPool();
 
+    private static int getNumFlusherPoolThreads() {
+        int numThreads = 100;
+        try {
+            String threads = System.getProperty("num_flusher_pool_threads", "100");
```

Comment:
The issue with using a Config for this is that readStormConfig inside the worker would read the system config, not the topology config, and would not let us override it on a per topology bases.  I will add in a config for the system default, but in the documents there indicate the system property that can be set to override it for a topology.